sos/plugins/__init__.py

```diff
@@ -88,40 +88,43 @@ def _node_type(st):
 
 
 def _file_is_compressed(path):
-    """Check if a file appears to be compressed
-
-    Return True if the file specified by path appears to be compressed,
-    or False otherwise by testing the file name extension against a
-    list of known file compression extentions.
+    """Check if a file appears to be compressed.
+
+    :param path: File path.
+    :returns: ``True`` if the file specified by path appears to be
+               compressed, or False otherwise by testing the file name
+               extension against a list of known file compression
+               extentions.
+    :returntype: bool
     """
     return path.endswith(('.gz', '.xz', '.bz', '.bz2'))
 
 
 class Plugin(object):
-    """ This is the base class for sosreport plugins. Plugins should subclass
+    """This is the base class for sosreport plugins. Plugins should subclass
     this and set the class variables where applicable.
 
-    plugin_name is a string returned by plugin.name(). If this is set to None
-    (the default) class\_.__name__.tolower() will be returned. Be sure to set
-    this if you are defining multiple plugins that do the same thing on
-    different platforms.
-
-    requires_root is a boolean that specifies whether or not sosreport should
-    execute this plugin as a super user.
-
-    version is a string representing the version of the plugin. This can be
-    useful for post-collection tooling.
-
-    packages (files) is an iterable of the names of packages (the paths
-    of files) to check for before running this plugin. If any of these packages
-    or files is found on the system, the default implementation of
-    check_enabled will return True.
-
-    profiles is an iterable of profile names that this plugin belongs to.
-    Whenever any of the profiles is selected on the command line the plugin
-    will be enabled (subject to normal check_enabled tests).
+    Attributes:
```

Comment:
Hi Sachin,

Unfortunately this class docstring is still using Google notation: *any* docstring that has "Attributes:" etc. labels is Google-style. The native Sphinx format uses RST.

Please [see here](https://thomas-cokelaer.info/tutorials/sphinx/_modules/template.html#MainClass1) for an example of a correct Sphinx-style class docstring using RST notation.